src/libraries/Common/src/System/Drawing/KnownColor.cs

```diff
@@ -164,6 +164,7 @@ enum KnownColor
         Plum,
         PowderBlue,
         Purple,
+        RebeccaPurple,
```

Comment:
You still need to add it to the ref, but with a new int value for the enum.

Reply:
Yep and you need to ensure the value in the source matches, the easiest way to do this is to move `RebeccaPurple` to the bottom here (it can't be alphabetical unless you explicitly number all values in here which is probably undesirable)

Reply:
Please move this to the end of the enum on this file.

Reply:
done. sorry i was late was busy